libc/src/__support/math/exp.h

```diff
@@ -234,7 +234,7 @@ static double set_exceptional(double x) {
 
 namespace math {
 
-static double exp(double x) {
+static constexpr double exp(double x) {
```

Comment:
this function should also be `LIBC_INLINE`
```suggestion
LIBC_INLINE static constexpr double exp(double x) {
```